docs/workflows.rst

```diff
@@ -60,15 +60,12 @@ single reference template (see `Longitudinal processing`_).
     as if they are unprocessed.
     In the case of brain-extracted (skull-stripped) T1w images, attempting to perform
     brain extraction again will often have poor results and may cause *fMRIPrep* to crash.
-    By default, *fMRIPrep* will attempt to detect these cases using a heuristic to check if the
-    T1w image is already masked.
+    *fMRIPrep* can attempt to detect these cases using a heuristic to check if the
+    T1w image is already masked. This must be explicitly requested with
+    ``---skull-strip-t1w auto``.
```

Comment:
We add linebreaks between sentences to keep the diffs focused in future PRs as well as the current one.

```suggestion
    *fMRIPrep* can attempt to detect these cases using a heuristic to check if the
    T1w image is already masked.
    This must be explicitly requested with ``---skull-strip-t1w auto``.
```